### PR TITLE
OBPIH-4211 Formatting changes to verifyRequest editPage

### PR DESCRIPTION
**Pull request by MagdaToczek**

### CODE REVIEW

**Review by pmuchowski — CHANGES_REQUESTED** [review]

File: src/js/components/stock-movement-wizard/request/EditPage.jsx [hunk 1]
```diff
@@ -105,6 +107,9 @@ const AD_HOCK_FIELDS = {
         attributes: {
           formatValue: value => (value ? (value.toLocaleString('en-US')) : value),
         },
+        getDynamicAttr: () => ({
```
[the remark this answers is not in the document]
Comment (MagdaToczek) [reply src/js/components/stock-movement-wizard/request/EditPage.jsx]: OK, corrected.

File: src/js/components/stock-movement-wizard/request/EditPage.jsx [hunk 2]
```diff
@@ -268,6 +268,7 @@ const AD_HOCK_FIELDS = {
         flexWidth: '1',
         attributes: {
           type: 'number',
+          numberField: true,
```
Comment (pmuchowski) [inline src/js/components/stock-movement-wizard/request/EditPage.jsx:271]: is this needed on Input field?

File: src/js/components/stock-movement-wizard/request/EditPage.jsx [hunk 3]
```diff
@@ -550,6 +551,7 @@ const STOCKLIST_FIELDS_PUSH_TYPE = {
...
```
Comment (pmuchowski) [inline src/js/components/stock-movement-wizard/request/EditPage.jsx:554]: as above

File: src/js/components/stock-movement-wizard/request/EditPage.jsx [hunk 4]
```diff
@@ -832,6 +834,7 @@ const STOCKLIST_FIELDS_PULL_TYPE = {
...
```
Comment (pmuchowski) [inline src/js/components/stock-movement-wizard/request/EditPage.jsx:837]: as above

File: src/js/components/stock-movement-wizard/request/EditPage.jsx (hunk 2 above)
Comment (MagdaToczek) [reply src/js/components/stock-movement-wizard/request/EditPage.jsx]: It is not needed.

File: src/js/components/stock-movement-wizard/request/EditPage.jsx (hunk 3 above)
Comment (MagdaToczek) [reply src/js/components/stock-movement-wizard/request/EditPage.jsx]: It is not needed.

File: src/js/components/stock-movement-wizard/request/EditPage.jsx (hunk 4 above)
Comment (MagdaToczek) [reply src/js/components/stock-movement-wizard/request/EditPage.jsx]: It is not needed.

**Review by pmuchowski — APPROVED** [review]

**Pull request merged**